pom.xml

```diff
@@ -224,7 +224,6 @@
                 <configuration>
                     <productVersion>${bamboo.version}</productVersion>
                     <productDataVersion>${bamboo.data.version}</productDataVersion>
-                    <productDataPath>../generated-test-resources.zip</productDataPath>
```

Comment:
I recall being unable to get a build without commenting out this line. Might that have to do with using `atlas-run`'s build side effect instead of using `atlas-package`?